src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStore.java

```diff
@@ -301,6 +307,18 @@ public String getDisplayName() {
         }
     }
 
+    /**
+     * create an S3 Bucket.
+     * @param name name of the S3 Bucket.
+     * @return return the Bucket created.
+     * @throws IOException in case of error.
+     */
+    public Bucket createS3Bucket(String name) throws IOException {
+        AWSStaticCredentialsProvider credentialsProvider = new AWSStaticCredentialsProvider(sessionCredentials());
+        AmazonS3 client = getAmazonS3ClientBuilder().withCredentials(credentialsProvider).build();
+        return client.createBucket(name);
```

Comment:
Does this actually throw `IOException` in case of trouble, or `RuntimeException`s?

I suppose it does not matter since the caller catches `Throwable`, but you might want to adjust the Javadoc to make it clearer.